utils/groovy/src/main/java/org/apache/brooklyn/util/groovy/GroovyJavaMethods.java

```diff
@@ -75,6 +76,20 @@
         }
     }
 
+    /**
+     * Alternative implementation of {@link ScriptBytecodeAdapter#isCase(Object, Object)}<br>
+     * Stripped down to work only for caseExpression of type <code>java.lang.Class</code>.<br>
```

Comment:
My point is that _if_ we don't need the functionality that `ScriptBytecodeAdapter` was giving us, then we should keep it as simple as possible; otherwise people will be looking at this code thinking "why is the call to this Groovy stuff needed?" when in fact it's not. It will be confusing. There's no need for it to do "isCase" any more, is there?